.github/workflows/python-versions.yml

```diff
@@ -1,5 +1,6 @@
 name: python versions
 on:
+  workflow_dispatch:
```

Comment:
Why though?

Reply:
I love why questions. It forces me to think! Thanks

The main purpose is to check if an issue gets fixed with a PR.
For instance, it's been few weeks since a regression was introduced and I want to check that a PR fixes it before it gets merged.

Scheduled workflows are lower priority and I don't want to make it to be triggered by a PR automatically.

Reply:
Perhaps it makes sense to trigger this on PRs for requirements changes. Specifically to make sure we're in good shape for 3.8. We could reduce the testing matrix to just Big Sur and Ubuntu to cost less CI.